packages/@okta/vuepress-theme-prose/util/attribution/cookies.js

```diff
@@ -0,0 +1,32 @@
+import Cookies from "cookies.js";
+
+const COOKIE_PREFIX = "_okta_";
+const HOST = "okta.com";
+const HOST_LENGTH = 8;
+
+function getDomain(host) {
+  return host.indexOf(HOST) == host.length - HOST_LENGTH && host;
```

Comment:
nit for readability:
```suggestion
  return (host.indexOf(HOST) == host.length - HOST_LENGTH) && host;
```